closure/goog/math/integer_test.js

```diff
@@ -1691,3 +1691,10 @@ function testBigShift() {
   assertEquals('-591981510028266767381876356163880091648',
                a.negate().shiftLeft(97).toString());
 }
+
+// Regression test for
+// https://github.com/google/closure-library/pull/498
+function testBase36toString() {
+  var x = goog.math.Integer.fromString('80000000',16);
+  assertEquals('zzzzzz', goog.math.Integer.fromString('zzzzzz', 36).toString(36));
```

Comment:
no, I explicitly checked and it was somewhere around 50, but I must have checked in an extremely wrong fashion because it's actually 82.